conf/messages.en

```diff
@@ -216,3 +216,31 @@ protectedMicroserviceAuth.checkYourAnswersLabel =  Protected Microservices shoul
 protectedMicroserviceAuth.error.required = Select yes if  Protected Microservices are being authenticated and authorised by default.
 protectedMicroserviceAuth.change.hidden = ProtectedMicroserviceAuth
 protectedMicroserviceAuth.hint = Internal Auth is an option for service-to-service authentication.
+
+#========================================
+#            Admin Services
+#========================================
+
+noPublicRoute.title = Does your service have Public Routing to the service in MDTP Frontend Routes?
+noPublicRoute.heading = Does your service have Public Routing to the service in MDTP Frontend Routes?
+noPublicRoute.checkYourAnswersLabel = Does your service have Public Routing to the service in MDTP Frontend Routes?
+noPublicRoute.error.required = Select yes if your admin service has a public routing to the service in the MDTP Frontend Routes
+noPublicRoute.change.hidden = NoPublicRoute
+
+strideOrVPN.title = Is your admin service only accessible either on the HMRC VPN or by using a Stride device?
+strideOrVPN.heading = Is your admin service only accessible either on the HMRC VPN or by using a Stride device?
+strideOrVPN.checkYourAnswersLabel = Is your admin service only accessible either on the HMRC VPN or by using a Stride device?
+strideOrVPN.error.required = Select yes if your admin service only accessible either on the HMRC VPN or by using a Stride device
+strideOrVPN.change.hidden = StrideOrVPN
+
+strideOrInternalAuth.title = Is access to this service restricted to Stride or an Internal Auth?
+strideOrInternalAuth.heading = Is access to this service restricted to Stride or an Internal Auth?
+strideOrInternalAuth.checkYourAnswersLabel = Is access to this service restricted to Stride or an Internal Auth?
+strideOrInternalAuth.error.required = Select yes if access to this service restricted to Stride or an Internal Auth
+strideOrInternalAuth.change.hidden = StrideOrInternalAuth
+
+accessProductionAdminFrontend.title = Has access to the Production Admin Service Frontend been specified in conjunction with your Risk Assessor?
```

Comment:
I think this one is really poorly worded in the actual PRA.

I think it should probably read something like:
"Have the admin service`s access routes and authorisation restrictions been agreed with your Risk Assessor?"